client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/EventShowingsContainer.tsx

```diff
@@ -0,0 +1,161 @@
+import {useEvent} from './EventProvider';
+import {createSubmitFunction} from './ShowingUtils';
+import {toDateStringFormat} from '../../Events/showingInputContainer_deprecated';
+
+export const EventShowingsContainer = () => {
+  const {
+    token,
+    setReloadShowing,
+    showingData,
+    eventData,
+    setEditing,
+    editing,
+    setPopUpProps,
+    showPopUp,
+  } = useEvent();
+  const [add, setAdd] = useState(false);
+  const [sortBy, setSortBy] = useState(0);
+  const onSuccessAddShowing = () => {
+    setEditing((edit) => !edit);
+    setPopUpProps('Success', 'Showing Successfully Created', true);
+    setReloadShowing((reload) => !reload);
+    setAdd((add) => !add);
+  };
+  const onError = async (event) => {
+    try {
+      const res = await event.json();
+      setPopUpProps('Failure', res.error, false);
+    } catch (error) {
+      setPopUpProps('Failure', 'Showing creation failed', false);
+    }
+  };
+
+  const getSortingFunction = () => {
+    switch (sortBy) {
+    case 1:
+      return (a, b) => a.eventinstanceid - b.eventinstanceid;
+    case 2:
+      return (a, b) => a.totalseats - b.totalseats;
+    case 3:
+      return (a, b) => a.availableseats - b.availableseats;
+    case 4:
+      return (a, b) =>
+        new Date(toDateStringFormat(b.eventdate)).getTime() -
+          new Date(toDateStringFormat(a.eventdate)).getTime();
+    case 5:
+      return (a, b) => b.eventinstanceid - a.eventinstanceid;
+    case 6:
+      return (a, b) => b.totalseats - a.totalseats;
+    case 7:
+      return (a, b) => b.availableseats - a.availableseats;
+    default:
+      return (a, b) =>
+        new Date(toDateStringFormat(a.eventdate)).getTime() -
+          new Date(toDateStringFormat(b.eventdate)).getTime();
+    }
+  };
+
+  return (
+    <div
+      className={
+        'bg-white flex flex-col mt-6 p-6 rounded-xl shadow-xl text-zinc-800'
+      }
+    >
+      <div className={'grid grid-cols-12 p-2 gap-2'}>
+        <div
+          className={
+            'col-span-12 min-[768px]:col-span-7 flex flex-row flex-wrap justify-center min-[768px]:justify-start gap-2'
+          }
+        >
+          <h2
+            className={
+              ' text-xl md:text-2xl text-center min-[768px]:text-start w-fit h-fit my-auto'
+            }
+          >
+            {eventData?.eventname} Showings
+          </h2>
+          <button
+            className={
+              'bg-green-500 hover:bg-green-700 w-fit h-fit font-bold disabled:bg-gray-500 text-white px-2 py-1 rounded-xl flex flex-row'
+            }
+            onClick={() => {
+              setAdd((add) => !add);
+              setEditing((edit) => !edit);
+            }}
+            disabled={!eventData || editing}
+            aria-label={'Add Showing'}
+          >
+            <svg
+              xmlns='http://www.w3.org/2000/svg'
+              fill='none'
+              viewBox='0 0 20 20'
+              strokeWidth='2'
```

Comment:
Minor change, but could you make this `strokeWidth='3'` to match the other button